README.md

```diff
@@ -4,24 +4,45 @@ Welcome to the [nearmap.com](nearmap.com) C# test. The purpose of this assignmen
 
 ## Background
 
-The source code that you are given is a very simple imitation of a name/value store. `DatabaseStore` is the canonical data store that takes a while (50 ms) to store and retrieve data. `DistributedCacheStore` is an interface to a distributed cache that takes much less time to turn around - 5 ms. This scenario is a simplified example of a typical high performance server cluster with a database, a distributed cache and multiple worker nodes.
+The source code that you are given is a very simple imitation of a key/value store:
+
+* `DatabaseStore` is the central store that takes a while (500ms) to store and retrieve data.
+* `DistributedCacheStore` is an interface to a distributed cache that takes much less time to turn around (100ms to store or retrieve).
+
+This scenario is a simplified example of a typical high performance server cluster with a database, a distributed cache and multiple worker nodes.
 
 ## Assumptions 
+
+After startup:
 
-* Data in the canonical store (database) never changes and can be cached forever.
-* Data is never written to the canonical store (database), apart from during the startup.
-* If `GetValue` method returns `null`, the request data item does not exist and will never exist.
-* The solution should start with the distributed cache empty.
-* Provided code can be modified at will.
-* The aim of the test is to produce the simplest working solution to the problem. As such you are at liberty to choose any .NET framework classes.
+* Data in `DatabaseStore` never changes and can be cached forever.
+* If `DatabaseStore.GetValue()` returns `null` for a key, the requested data item does not exist and will never exist.
+* `DistributedCacheStore` is initially empty.
 
 ## Task
+
+Complete the 2 parts below & submit the solution. 
+If the solution is incomplete, please state what hasn't been finished and outline how you are planning on solving it.
+
+* Provided code can be modified at will.

```

Comment:
"Provided code can be modified at will, including any improvements you wish to make"?